taskt/Core/Automation/Commands/Data Commands/MathCalculationCommand.cs

```diff
@@ -1,127 +1,120 @@
-using System.Windows.Forms;
-using taskt.UI.Forms;
-using taskt.UI.CustomControls;
 using System.Data;
+using System.Windows.Forms;
+using System.Xml.Serialization;
+using taskt.Core.Automation.Attributes.ClassAttributes;
+using taskt.Core.Automation.Attributes.PropertyAttributes;
 using taskt.Core.Utilities.CommonUtilities;
+using taskt.UI.CustomControls;
+using taskt.UI.Forms;
 
 namespace taskt.Core.Automation.Commands
 {
     [Serializable]
-    [Attributes.ClassAttributes.Group("Data Commands")]
-    [Attributes.ClassAttributes.Description("This command allows you to perform a math calculation and apply it to a variable.")]
-    [Attributes.ClassAttributes.UsesDescription("Use this command when you want to perform a math calculation.")]
-    [Attributes.ClassAttributes.ImplementationDescription("")]
+    [Group("Data Commands")]
+    [Description("This command performs a math calculation and saves the result in a variable.")]
     public class MathCalculationCommand : ScriptCommand
     {
         [XmlAttribute]
-        [Attributes.PropertyAttributes.PropertyDescription("Please supply the input to be computed (ex. (2 + 1)")]
-        [Attributes.PropertyAttributes.PropertyUIHelper(Attributes.PropertyAttributes.PropertyUIHelper.UIAdditionalHelperType.ShowVariableHelper)]
-        [Attributes.PropertyAttributes.InputSpecification("Specify either text or a variable that contains valid math.")]
-        [Attributes.PropertyAttributes.SampleUsage("")]
-        [Attributes.PropertyAttributes.Remarks("You can use known numbers or variables.")]
-        public string v_InputValue { get; set; }
+        [PropertyDescription("Math Expression")]
+        [InputSpecification("Specify either text or a variable that contains valid math expression.")]
+        [SampleUsage("(2 + 5) * 3 || ({vNumber1} + {vNumber2}) * {vNumber3}")]
+        [Remarks("You can use known numbers or variables.")]
+        [PropertyUIHelper(PropertyUIHelper.UIAdditionalHelperType.ShowVariableHelper)]
+        public string v_MathExpression { get; set; }
 
         [XmlAttribute]
-        [Attributes.PropertyAttributes.PropertyDescription("Optional - Indicate Thousand Seperator")]
-        [Attributes.PropertyAttributes.PropertyUIHelper(Attributes.PropertyAttributes.PropertyUIHelper.UIAdditionalHelperType.ShowVariableHelper)]
-        [Attributes.PropertyAttributes.InputSpecification("Enter the seperator used to identify decimal places")]
-        [Attributes.PropertyAttributes.SampleUsage("")]
-        [Attributes.PropertyAttributes.Remarks("Typically a comma or a decimal point (period)")]
-        public string v_ThousandSeperator { get; set; }
+        [PropertyDescription("Thousand Separator (Optional)")]
+        [InputSpecification("Specify the seperator used to identify decimal places.")]
+        [SampleUsage(", || . || {vThousandSeparator}")]
+        [Remarks("Typically a comma or a decimal point (period), like ',' is a thousand separator in 100,000.")]
```

Comment:
Change to "like in 100,000, ',' is a thousand separator."

Reply:
done